Source/JavaScriptCore/API/JSManagedValue.mm

```diff
@@ -85,7 +85,7 @@ - (instancetype)initWithValue:(JSValue *)value
     if (!value)
         return self;
 
-    JSC::JSGlobalObject* globalObject = toJS([value.context JSGlobalContextRef]);
+    JSC::JSGlobalObject *globalObject = toJS(value.context.JSGlobalContextRef);
```

Comment:
This is not correct. Our convention is `*` on left hand side for non Objective-C class. And JSGlobalObject* is not Objective-C class.